blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobStoreIntegrationTest.java

```diff
@@ -531,6 +532,9 @@ protected Module createHttpModule() {
    }
 
    protected void awaitConsistency() {
+      if (!initialized) {
```

Comment:
This seems magical.  Instead should we pass a flag to `createContainerAndEnsureEmpty` which controls the call to `awaitConsistency`?